src/components/Footer.test.js

```diff
@@ -0,0 +1,18 @@
+// import App from './App';
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import Footer from './footer'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe('When App renders', () => {
+  let renderedFooter;
+  beforeEach(() => {
+    renderedFooter = shallow(<Footer />);
+  });
+  it('displays Footer', () => {
+    const footer = renderedFooter.find('h5');
+    expect(footer.length).toEqual(1)
```

Comment:
for more stringent tests you can also assert on the text expected to be seen within the footer.
```suggestion
    expect(footer.length).toEqual(1)
    expect(footer.text()).toEqual('Brought to you by Steph and Elyse from Learn Academy!')
```